src/pages/transactions/create/Create.tsx

```diff
@@ -85,33 +90,43 @@ export function Create() {
 
     setErrors(undefined);
 
-    setIsSaving(true);
-
-    toast.processing();
-
-    try {
-      await request('POST', endpoint('/api/v1/bank_transactions'), {
-        ...transaction,
-        amount: Number(transaction?.amount),
-        base_type:
-          transaction?.base_type === TransactionType.Deposit
-            ? ApiTransactionType.Credit
-            : ApiTransactionType.Debit,
-      });
-      toast.success(t('created_transaction'));
-      setIsSaving(false);
-      navigate(route('/transactions'));
-    } catch (error) {
-      setIsSaving(false);
-      const axiosError = error as AxiosError;
-      console.error(axiosError);
-
-      if (axiosError?.response?.status === 422) {
-        setErrors(axiosError?.response?.data?.errors);
-        toast.dismiss();
-      } else {
-        toast.error(t('error_title'));
+    if (transaction?.bank_integration_id) {
+      setIsSaving(true);
+
+      toast.processing();
+
+      try {
+        await request('POST', endpoint('/api/v1/bank_transactions'), {
+          ...transaction,
+          amount: Number(transaction?.amount),
+          base_type:
+            transaction?.base_type === TransactionType.Deposit
+              ? ApiTransactionType.Credit
+              : ApiTransactionType.Debit,
+        });
+        toast.success(t('created_transaction'));
+        setIsSaving(false);
+        navigate('/transactions');
+      } catch (error) {
+        setIsSaving(false);
+        const axiosError = error as AxiosError;
+        console.log(axiosError);
+        console.error(axiosError);
+
+        if (axiosError?.response?.status === 422) {
+          setErrors(axiosError?.response?.data?.errors);
+          toast.dismiss();
+        } else {
+          toast.error(t('error_title'));
```

Comment:
toast is translatable. Just pass plain string, please:

```ts
toast.error('error_title')
```

or in this case:

```ts
toast.error()
```

Reply:
changed